x/exchange/spec/01_concepts.md

```diff
@@ -109,6 +118,23 @@ E.g. If an order's funds are in a sanctioned account, settlement of that order w
 Or, if a marker has required attributes, but the recipient does not have those attributes, settlement will fail.
 
 
+### Commitment Settlement
+
+A market can move funds committed to it by using the [MarketCommitmentSettle](03_messages.md#marketcommitmentsettle) endpoint.
+
+During commitment settlement:
+
+1. Holds are released on the funds to be transferred.
+2. The funds are transferred
```

Comment:
You are missing a period at the end of the sentence.

Reply:
Fixed.